docs/content/design/segments.md

```diff
@@ -52,4 +165,10 @@ Sharding Data to Create Segments
 
 ### Sharding Data by Dimension
```

Comment:
In general, we "partition" the data source by time and then we "shard" the partitions by some other dimension/set of dimensions.